### DISCUSSION TITLE
Need help triaging Issues/PRs?

### DISCUSSION THREAD

**Discussion by mattcen** [hakimel/reveal.js#3438] [General]
Hi @hakimel,

I see there are *lots* of open Issues, many of which may be obsolete, resolved, incorrectly tagged, or better as Discussions. Likewise, I stumbled across two near-identical PRs before (and tagged them as such).

Is there any way we can help you tidy up some of these to make it easier to find things when searching?

I for one didn't realise Discussions were being used because there were so many Discussion-shaped Issues, I assumed that was where all the chat was happening.

**Comment by mattcen** [comment]
Reading the above, I can see it might come across as a criticism, which is not at all how I intend it; it's a genuine offer to help.

I love Reveal.js as a tool, and have been using it for almost 10 years. I was just digging through to try to work out how to use some new-ish features in one of my presentations (and integrate it into my presentation template), and had a bunch of issues to wade through to find if what I sought was possible, so I want to help save others some time in future.

**Comment by hakimel** [comment]
Hi @mattcen! Don't worry about it, your message didn't come across as criticism. It's true that there's a lot of work to be done organizing issues and PRs. Most of the existing issues were opened before Discussions were enabled and it'd be great to move questions over to there.

If you're open to helping even if it's just an issue here and there it'd be much appreciated. Thanks!

**Reply by mattcen** [reply]
Sounds great! Thanks for the opportunity! I've just accepted the invite, and will do what I can when I have some time on my hands! :)

**Comment by t-fritsch** [comment]
I also just found out that discussions were open here, and as @mattcen I'd be happy to help (funny thing is I'm in the same situation as you @mattcen : using reveal since a lot of time - 2016, and am currently upgrading my presentations to the latest version, searching for new features, new plugins, tuning reveal to match my needs, etc.).

I don't usually have a lot of time but I'm on holiday and some time to help 😄

**Comment by hakimel** [comment]
Appreciate all of the PRs @t-fritsch!

I'd love the help if you want to close out issue or convert any of them to discussions. Just sent a collaborator invite.

**Reply by t-fritsch** [reply]
Yes I just accepted the invitation, thanks a lot for your trust

I'll do the best I can, this project is really great ❤️

**Comment by Martinomagnifico** [comment]
Thanks for all the PRs, tagging and looking into issues, @mattcen & @t-fritsch ! Really appreciated! Same situation here by the way: using Reveal a lot and wanted to help a bit.

**Reply by t-fritsch** [reply]
you're welcome, thank you for the great plugins you make !! 
I'm a big fan of Simplemenu plugin since I discovered it last week 😄
